### DISCUSSION TITLE
Import Pole Figure

### DISCUSSION THREAD

**Discussion by Andre-calazans** [mtex-toolbox/mtex#547] [Ask Anything] [answered]
Hi,

I have been trying to import my pole figures. My pole figure's file is column1= polar angle, column2=polar angle and column3=intensity. However, MTEX only allows me to import the column2 as azimuth angle. When I plot my pole figure, the figure is different from the expected pole figure (see below). What should I do to preserve polar angle during the importation?

![image](https://user-images.githubusercontent.com/73460174/97401242-6b55b800-192b-11eb-8dfc-435556591035.png)
[METX doubt.pptx](https://github.com/mtex-toolbox/mtex/files/5450094/METX.doubt.pptx)

**Comment by kilir** [comment, marked as answer]
Hi,
it might be a typo, but "column1= polar angle, column2=polar angle and column3=intensity" does not work. You can change the column interpretation in the wizard by just clicking on the fields in the area you highlighted.

Cheers,
Rüdiger

**Reply by Andre-calazans** [reply]
Hi Rudiger,

Thank you for your feedback!
If I change the column interpretation in the wizard (polar angle, polar angle, intensity) it does not allow me to click finish. What do you suggest? If I click azimuth angle, it will import wrongly as I mentioned before.
Cheers,
Andre

**Reply by kilir** [reply]
Maybe it is a misconception, but you like to specify points in 2D by just providing coordinates of the same type, if I read you correctly.
Neither polar-polar nor azimuth-azimuth makes a sense (that's like specifying a point in just by two x-coordinates or two y-coordinates), that's why you can't click finish in the wizard.

From looking at your pf, either, you selected the wrong columns or you need to change your coordinate system.

Cheers,
Rüdiger

**Reply by Andre-calazans** [reply]
OK!
Thank you!
Andre
